independent-publisher-connectors/Zoom Meetings/readme.md

```diff
@@ -0,0 +1,26 @@
+# Zoom Connector
+Zoom is widely used for Virtual Meetings. Using this Connector you can Create Meetings, Get Meetings and Meeting Details (Individually). In some scenarios we want to perform certain actions Ex: Add Meetings into Excel and many more. In this process Zoom Connector will gets the work done.
+
+## Prerequisites
+You will need the following to proceed:
+•	A Microsoft Power Apps or Power Automate plan with custom connector feature
+•	A Zoom Account

```

Comment:
@A-Deekshith Please add a section called "Obtaining credentials and Setting up the application" and provide the instructions accordingly which would help us to setup an app to validate your connector.

Reply:
@A-Deekshith, Please reach-out to me if you need any support on this connector. Wishing you all the best.

Reply:
@sriyen-msft @nk-gears Sir, I had done all the changes that you suggested. I request you to review the Files.